### PR TITLE
Add configuration example files

### PR DESCRIPTION
**Pull request by fedepaol**
At the same time, fixing the type of the rawconfig field because it renders awfully as []byte.

### CODE REVIEW

File: config/samples/advertise_two_neighs_sameconfig.yaml
Comment (oribon) [inline config/samples/advertise_two_neighs_sameconfig.yaml:1]: general comment for all files: defaults to 0/{}/restricted mode without prefixes/irrelevant metadata/.. can be removed

Comment (oribon) [inline config/samples/advertise_two_neighs_sameconfig.yaml:1]: nit in name: sameconfig -> one_config

File: config/samples/advertise_two_neighs_split.yaml
Comment (oribon) [inline config/samples/advertise_two_neighs_split.yaml:1]: split -> two_configs

File: config/samples/advertise_two_neighs_sameconfig.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  bgp:
+    routers:
+    - asn: 64512
+      neighbors:
+      - address: 172.30.0.3
+        asn: 4200000000
+        ebgpMultiHop: true
+        holdTime: 0s
+        keepaliveTime: 0s
+        password:
```
Comment (oribon) [inline config/samples/advertise_two_neighs_sameconfig.yaml:15]: wdyt about adding the secret as well?

File: config/samples/node_selector.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        port: 180
+        toAdvertise:
+          allowed:
+            mode: all
+        toReceive:
+          allowed:
+            mode: all
+  nodeSelector:
+    labelSelector:
+      "foo": "bar"
```
Comment (oribon) [inline config/samples/node_selector.yaml:25]: "foo" -> foo

File: config/samples/advertise_two_neighs_sameconfig.yaml (hunk 1 above)
Comment (fedepaol) [reply config/samples/advertise_two_neighs_sameconfig.yaml:15]: made things even simpler, removed passwords from all and adding a _with_password custom example

File: config/samples/neighbor_with_password.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
+kind: FRRConfiguration
```
Comment (oribon) [inline config/samples/neighbor_with_password.yaml:1]: same

File: config/samples/advertise_two_neighs_two_configs.yaml [hunk 4]
```diff
@@ -0,0 +1,43 @@
+apiVersion: frrk8s.metallb.io/v1beta1
+kind: FRRConfiguration
+metadata:
+  name: test-0
+  namespace: default
```
Comment (oribon) [inline config/samples/advertise_two_neighs_two_configs.yaml:5]: change "default" to "frr-k8s-system" everywhere?

**Review by oribon — APPROVED** [review]
small nits, other than that lgtm

File: config/samples/advertise_two_neighs_two_configs.yaml (hunk 4 above)
Comment (fedepaol) [reply config/samples/advertise_two_neighs_two_configs.yaml]: done

**Pull request merged**